### ISSUE TITLE
device name command does not change the collar name

### ISSUE THREAD

**Issue by SilkieSabra** [OpenCollarTeam/OpenCollar#672]
Salt reported this to me, she says it worked on her collars in 8.0 but not in 8.1
I tested it on a stock collar (rezzed).  The chat command [prefix] device name [newname] delivered a message that says "The device name is now set to: OpenCollar" which was the original name.  After changing the name of the collar in the name field, the command repeated the new name but not uniformly unless the description field was also changed manually.  

We need to be able to change the device name for chat messages from the collar without having to alter the Description field.  The chat command should do that.

**Comment by Medea-Destiny** [comment]
@SilkieSabra Can you check that the device name isn't in fact changing when you set it with the chat command? Ignore the "device name is not set to:" response, how about messages that come through after that?

The "device name [name]" command is handled in the usercommand() function in oc_api, as follows: 
```
llMessageLinked(LINK_SET, LM_SETTING_SAVE, "global_devicename="+sChangevalue,"");
llMessageLinked(LINK_SET, NOTIFY, "0The device name is now set to: %DEVICENAME%", kID);
```

It sends an LM_SETTING_SAVE and  a NOTIFY one after the other; the incorrect response you reported would likely be caused by the fact that there's no delay between these two linked messages, so the dialog script would react to the NOTIFY with the current %DEVICENAME% as it would not yet have received the new setting back as an LM_SETTING_RESPONSE from the oc_settings script. If it is only this message failing that's the problem it could be fixed by dropping in say llSleep(0.4); between those two lines.

**Comment by Medea-Destiny** [comment]
I have confirmed that the device name does change and it is indeed just the notification message failing as I guessed. The fix I proposed works.
